### PR TITLE
Adding Cassandra

### PR DESCRIPTION
**Pull request by Kjames5269**
Test cases coming soon^tm

Download cassandra and do things 🤷‍♂

### CODE REVIEW

**Comment by vinamartin** [comment]
Add a README comment that you have to spin up a cassandra for the time being to test.

File: build.gradle [hunk 1]
```diff
@@ -27,7 +28,13 @@ dependencies {
 		force = true
 		because 'CVE-2019-0232 in the transitive tomcat-embed-core dependency'
 	}
-    implementation('org.springframework.data:spring-data-cassandra')
```
Comment (jhunzik) [inline build.gradle]: Do we run the formatter on Groovy files? Some of the tabbing looks inconsistent.

File: src/test/java/com/connexta/multiintstore/storage/persistence/MockDB.java [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+import com.connexta.multiintstore.storage.persistence.models.Storable;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Optional;
+import java.util.UUID;
+import org.mockito.invocation.InvocationOnMock;
+
+/**
+ * Helper class for mocking out the Metadata Repository. This class will be usable by anyone
+ * extending a the {@link org.springframework.data.repository.CrudRepository}
```
Comment (jhunzik) [inline src/test/java/com/connexta/multiintstore/storage/persistence/MockDB.java]: ✏️ `extending the`

File: build.gradle (hunk 1 above)
Comment (Kjames5269) [reply build.gradle]: I don't think we do... I can go through and try to fix it though

File: src/test/java/com/connexta/multiintstore/storage/persistence/MockDB.java (hunk 2 above)
Comment (Kjames5269) [reply src/test/java/com/connexta/multiintstore/storage/persistence/MockDB.java]: English is hard. Nice catch

**Comment by Kjames5269** [comment]
build now

**Comment by Kjames5269** [comment]
build now

**Comment by Kjames5269** [comment]
build now

**Pull request merged**